zndraw/server/events.py

```diff
@@ -340,21 +365,43 @@ def scene_pause(data):
 @io.on("modifier:register")
 def modifier_register(data):
     data["token"] = session["token"]
+    if "modifier" not in app.config["PER-TOKEN-DATA"][session["token"]]:
+        # this is a dict for the pyclient, but it has to be for every webclient ...
+        app.config["PER-TOKEN-DATA"][session["token"]]["modifier"] = {}
```

Comment:
This should be a function. We use it in multiple places, would be better to have `def add_modifier_to_config(session_token):...`